demo/docker/nginx/nginx.conf

```diff
@@ -14,6 +14,9 @@ http {
       proxy_pass http://akhq:8080/akhq/;
     }
 
+    location /ui/ {
+      proxy_pass http://akhq:8080/akhq/ui/;
+    }
```

Comment:
If we had Apache we could handle relative redirects with `ProxyPassReverse`, but nginx does not have it.
But I found on the Web https://www.nginx.com/resources/wiki/start/topics/examples/likeapache/
Would it be any good in this case?